docs/source/install.rst

```diff
@@ -12,15 +12,19 @@ Conda
 
 Dask is installed by default in `Anaconda <https://www.anaconda.com/download/>`_::
 
-You can update Dask using `conda <https://www.anaconda.com/download/>`_::
+You can update Dask using the `conda <https://www.anaconda.com/download/>`_: command:
 
-    conda install dask
+*   ``conda install dask``               : Installs dask from the default channel (recommended)
+*   ``conda install -c conda-forge dask``: Installs dask from `conda-forge <https://conda-forge.github.io/>`_.
 
-This installs Dask and all common dependencies, including Pandas and NumPy.
+This installs Dask and **all** common dependencies, including Pandas and NumPy.
 
-Dask packages are maintained both on the default channel and on and
-`conda-forge <https://conda-forge.github.io/>`_.
+Optionally, you can obtain a no frills dask installation using:
 
+*   ``conda install dask-core``                : Installs dask-core from the default channel (recommended)
+*   ``conda install -c conda-forge dask-core`` : Installs dask-core from `conda-forge <https://conda-forge.github.io/>`_.
+    
+This will install a minimal set of dependencies required to run dask, similar to (but not exactly the same as) ``pip install dask`` below. See https://github.com/conda-forge/dask-feedstock/issues/22 for more information on the large number of dependencies `conda install dask` would pull.
```

Comment:
I don't think that we should link to github here.  If there is a need to describe the situation more fully then we should do so here as the central source of information.

Reply:
Sure, people should be able to find out more through a simple git blame/web search. It did take me a while to track down what the situation was (I went from `pip install dask` to `conda install dask` and found myself with a lot of unnecessary bloat), but I guess this PR change will solve that. I'll remove the reference link.